composer.json

```diff
@@ -9,7 +9,7 @@
     },
     "minimum-stability": "stable",
     "require": {
-        "php": ">=5.4,<8.0-dev",
+        "php": "~5.6.0",
```

Comment:
In that case, you ought to update the Travis build file to upgrade the PHP requirement to 7.x while testing. Otherwise the dependencies won't install.